docs/pages/access-controls/device-trust/device-management.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Manage Trusted Devices (Preview)
+description: Learn how to manage Trusted Devices
+---
+
+<Admonition type="warning" title="Preview Note: Supported Device Type">
+The device trust preview works on macOS devices. Support for other operating
+systems and access features is planned for upcoming Teleport versions.
+</Admonition>
+
+### Prerequisites
+
+(!docs/pages/includes/no-oss-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/device-trust/prereqs.mdx!)
+
+## Register a trusted device
+
+The `tctl` tool is used to manage the device inventory. A device admin is
+responsible for managing devices, adding new devices to the inventory and
+removing devices that are no longer in use.
+
+<Admonition type="tip" title="Self enrollment: v13.3.5+">
+  Users with the preset `editor` or `device-admin` role (since v13.3.6)
+  can register and enroll their device in a single step with the following command:
+  ```code
+  $ tsh device enroll --current-device
+  ```
+</Admonition>
+
+To register a device, you first need to determine its serial number.
+
+Replace the `$SERIAL` below with your macOS device serial number (visible under Apple ->
+"About This Mac" -> "Serial number", or use the command below).
+
+```code
+$ SERIAL="$(ioreg -c IOPlatformExpertDevice -d 2 | grep -i IOPlatformSerialNumber | awk -F'"' '{print $4}')"
+$ tctl devices add --os=macos --asset-tag="$SERIAL"
+Device (=devicetrust.asset_tag=)/macOS added to the inventory
+```
+
+View registered devices:
+
+```code
+$ tctl devices ls
+Asset Tag    OS    Enroll Status Device ID
+------------ ----- ------------- ------------------------------------
+(=devicetrust.asset_tag=) macOS not enrolled  (=devicetrust.device_id=)
+```
+
+<Admonition type="warning" title="Device Role">
+  For clusters created using Teleport v12 or newer, the preset `editor` role has
+  the necessary permissions to manage devices. For clusters created before v12, you may want
+  to create a dedicated [`device-admin`](#dedicated-device-admin-role) role.
+</Admonition>
+
+## Create a device enrollment token
+
+A registered device becomes a trusted device after it goes through the
+enrollment ceremony. To enroll the device, a device enrollment token is
+necessary. The token is created by a device admin and sent to the person
+performing the enrollment off-band (for example, via a corporate chat).
+
+To create an enrollment token run the command below, where `--asset-tag` is
+the serial number of the device we want to enroll:
+
+```code
+$ tctl devices enroll --asset-tag="(=devicetrust.asset_tag=)"
+Run the command below on device "(=devicetrust.asset_tag=)" to enroll it:
+tsh device enroll --token=(=devicetrust.enroll_token=)
+```
+
+## Enroll a trusted device
+
+To perform the enrollment ceremony, using the device specified above, type the
+command printed by `tctl devices enroll`:
```

Comment:
Does `tctl devices enroll` exist in v12?

Reply:
It does. It creates enrollment token: 
```
$ tctl devices enroll --asset-tag="C00AA0AAAA0A"
Run the command below on device "C00AA0AAAA0A" to enroll it:
tsh device enroll --token=AAAAAAAAAAAAAAAAAAAAAAAA-this-is-an-example
```